storage/onpremise/mongodb-sharded/secrets.tf

```diff
@@ -0,0 +1,47 @@
+data "kubernetes_secret" "mongodb_credentials" {
+  metadata {
+    name      = helm_release.mongodb.name
+    namespace = var.namespace
+  }
+}
+
+resource "kubernetes_secret" "mongodb_admin" {
```

Comment:
Why those two secrets `mongodb_admin` and `mongodb_user` have different keys?

Reply:
`mongodb_user` secret is now passed as env variable to the chart. It will be used for manual connections, mainly for debugging purposes I think, since Core now gets admin credentials.